example/ssd/detect/detector.py

```diff
@@ -43,6 +43,7 @@ class Detector(object):
     ctx : mx.ctx
         device to use, if None, use mx.cpu() as default context
     """
+    CLASS = 0
```

Comment:
Is this convention used elsewhere? All caps, word 'CLASS', etc?
Is it used somewhere?  Can you name it something that isn't a keyword else and not in caps?

Reply:
https://www.python.org/dev/peps/pep-0008/#constants

Reply:
KLASS?

Reply:
"Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include MAX_OVERFLOW and TOTAL."

This is at class-level, right?  Would a derivative not want to override the value?
Is this done elsewhere? I haven't seen thyis done within classes elsewhere, but maybe I'm wrong?

Reply:
this is at class level, is the equivalent of a static variable in C++. A variable that is owned by the class itself, instead of the instance. Since it's a constant is in caps.  I don't see a problem, other than perhaps the naming. So the variable refers to which class the object belongs to. class is a reserved identifier in python as you mention, and you indicate that having a reserved word just differing by the case might not be good. I don't see a problem, in any case we could rename it to KLASS or CLS as is done usually in python. Maybe @zhreshold  has additional comments anyway.

Reply:
I understand that you mean it for class-level.  
Again, question:

"Would a derivative not want to override the value?
Is this done elsewhere (all-caps constant within a class)? I haven't seen this done within classes elsewhere, but maybe I'm wrong?"

Reply:
I think it's better to use cls_index=0 as default argument when instantiating rather than class level static variable